src/folderview.h

```diff
@@ -130,7 +130,7 @@ class LIBFM_QT_API FolderView : public QWidget {
     void setCustomColumnWidths(const QList<int> &widths);
 
     QList<int> getHiddenColumns() const {
-        return hiddenColumns_.toList();
+        return hiddenColumns_.values();
```

Comment:
The same as above.

Reply:
Ok. It's just that the compiler suggested me to use values :)
I'll ifdef it then